tests/utilities/helper.py

```diff
@@ -7,8 +7,8 @@
 import random
 import string
 
-DEVOPS_CLI_TEST_ORGANIZATION = 'https://AzureDevOpsCliTest.visualstudio.com'
-DEVOPS_CLI_TEST_PAT_TOKEN = 'vj3ep2pg3fo6vxsklkwvkiy23dkbyynmfpg4vb66xniwr23zylla'
+DEVOPS_CLI_TEST_ORGANIZATION = 'https://dev.azure.com/azuredevopsclitest'
```

Comment:
may be for some other day
we have this constant across tests
so say I record my test against https://dev.azure.com/gsaral
then it will fail in recordings? because URI will not match (my assumption)

Reply:
Hmm.. noticed it while recording for the policy test after merge.